feugy/melodie
# docs: fixes missing github icon on site

### Reproduction steps

1. browse to https://feugy.github.io/melodie/
   > the top right corner should include a white github icon, like this one:

![image](https://user-images.githubusercontent.com/186268/147793662-c5e23739-d303-4d38-a16f-249d9e37925f.png)

- [x] docs: fixes missing github icon on site

### How to test?

1. On your machine, build the site:
   ```sh
   npm -w apps/site run build
   ```
1. Then serve the web site
   ```sh
   npm -w apps/site run serve
   ```
1. finally, browse to the site: http://localhost:3000/melodie

Please note that the icon path is `/melodie/logos/github.png`

### Notes to reviewers

It's a tricky one: `/logos/github.png` will also be served locally, hence why this regression made it to production. 
I couldn't find a way to configure vite so it only serves `/melodie/logos/`
